src/com/nutrons/FRamework/input/InputFactory.java

```diff
@@ -30,16 +29,12 @@ public static InputFactory instance() {
         return InputFactory.instance;
     }
 
-    public <T> Flowable<T> toFlow(Supplier<T> s, long period, TimeUnit unit) {
-        return Flowable.interval(period, unit).map((x) -> s.get());
-    }
-
     private Joystick lazyJoy(int instance) {
         if (!this.joysticks.containsKey(instance)) {
             Joystick j = new Joystick(instance);
             this.joysticks.put(instance, j);
-            this.joysticksX.put(instance, this.toFlow(() -> j.getAxis(Joystick.AxisType.kX), 100, TimeUnit.MILLISECONDS));
-            this.joysticksY.put(instance, this.toFlow(() -> j.getAxis(Joystick.AxisType.kY), 100, TimeUnit.MILLISECONDS));
+            this.joysticksX.put(instance, this.toFlow(() -> j.getAxis(Joystick.AxisType.kX)));
```

Comment:
Make put a method that operates on the instance.   Since that's the only instance of the class, just call a method directly